bootstrap/roles/appliance-build.bootstrap/tasks/main.yml

```diff
@@ -29,6 +29,8 @@
     path: "{{ toplevel.stdout }}/docker"
     name: appliance-build
 
+- include: install_zfs.yml
```

Comment:
I would prefer just to add "zfsutils-linux" to list of packages we're adding a few lines above.. do you think it's better to add a new file, and include it here?

Reply:
My understanding (admittedly limited) of ansible is that it's best to split logically separate dependencies into separate files and include as necessary. If that's not the convention, or overkill for this small of a dependency list, I'm fine including it in the same file as well.

Reply:
I think it's overkill in this case, since we're just installing a single package, and we already have some logic to do that for other packages above. But I'll defer to your judgement here; if you think a new file is best, that's OK with me.